app/ddm/datasets/page.tsx

```diff
@@ -3,36 +3,35 @@
 import * as React from 'react';
 import * as Utilities from '@common/utilities';
 
-import styles from './Datasets.module.scss';
+import styles from './page.module.scss';
```

Comment:
No need to use `./` syntax when you have the `@` shorthand

Reply:
fixed. also added `@ddm/*` mapping to `/app/ddm/*`